index.html

```diff
@@ -48,148 +48,3 @@ <h2 >Here's a time line of Dr. Borlaug's life:</h2>
-              >Rejects a 100% salary increase from Dupont, leaves behind his
-              pregnant wife, and flies to Mexico to head a new plant pathology
-              program. Over the next 16 years, his team breeds 6,000 different
-              strains of disease resistent wheat - including different varieties
-              for each major climate on Earth.
-            </span>
-          </li>
-          <li>
-            <span class="date">1945</span> -
-            <span class="detail"
-              >Discovers a way to grown wheat twice each season, doubling wheat
-              yields
-            </span>
-          </li>
-          <li>
-            <span class="date">1953</span> -
-            <span class="detail"
-              >crosses a short, sturdy dwarf breed of wheat with a high-yeidling
-              American breed, creating a strain that responds well to
-              fertilizer. It goes on to provide 95% of Mexico's wheat</span
-            >
-          </li>
-          <li>
-            <span class="date">1962</span> -
-            <span class="detail"
-              >Visits Delhi and brings his high-yielding strains of wheat to the
-              Indian subcontinent in time to help mitigate mass starvation due
-              to a rapidly expanding population
-            </span>
-          </li>
-          <li>
-            <span class="date">1970</span> -
-            <span class="detail">receives the Nobel Peace Prize</span>
-          </li>
-          <li>
-            <span class="date">1983</span> -
-            <span class="detail"
-              >helps seven African countries dramatically increase their maize
-              and sorghum yields
-            </span>
-          </li>
-          <li>
-            <span class="date">1984</span> -
-            <span class="detail"
-              >becomes a distinguished professor at Texas A&M University
-            </span>
-          </li>
-          <li>
-            <span class="date">2005</span> -
-            <span class="detail"
-              >states "we will have to double the world food supply by 2050."
-              Argues that genetically modified crops are the only way we can
-              meet the demand, as we run out of arable land. Says that GM crops
-              are not inherently dangerous because "we've been genetically
-              modifying plants and animals for a long time. Long before we
-              called it science, people were selecting the best breeds."
-            </span>
-          </li>
-          <li>
-            <span class="date">2009</span> -
-            <span class="detail">dies at the age of 95.</span>
-          </li>
-        </ul>
-        <div class="tribute">
-          <p>
-            "Borlaug's life and achievement are testimony to the far-reaching
-            contribution that one man's towering intellect, persistence and
-            scientific vision can make to human peace and progress."
```

Comment:
- [ ] Consider closing this `p` tag at the end of your text :8ball:

Reply:
same here, Let's make this comment better by adding a reason for requesting the change😉 don't forget, the reasons may be clear to you but not to the recipient😉 Kindly edit the the comment and add a polite and descriprtive reason for your changes👍